app/core/utils.js

```diff
@@ -2373,6 +2373,21 @@ const getUserTimeZone = function (user) {
   }
 }
 
+const shouldShowAiBotHelp = function (aceConfig) {
+  if (aceConfig.levelChat != 'none') {
+    if(me.isAdmin()) {
+      return true
+    }
+    else if(me.isHomeUser() && me.getLevelChatExperimentValue() == 'beta') {
```

Comment:
why is else if on next line? 🤔

Reply:
lol interesting. i think eslint would fix it but seems not?